images/checkout.sh

```diff
@@ -34,13 +47,47 @@ git submodule init
 git submodule update
 
 # Print out the commit so we can tell from logs what we checked out.
-echo Repo is at `git describe --tags --always --dirty`
+echo ${REPO_DIR} is at `git describe --tags --always --dirty`
 git submodule
 git status
 
+# Check out any extra repos.
```

Comment:
would it be easier with GitPython ?

Reply:
I'm open to porting this script to python but we should keep it simple because its used to bootstrap things by checking out source code.